src/index.html

```diff
@@ -1,16 +1,52 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
   <head>
-    <meta charset="UTF-8">
+    <meta charset="UTF-8" >
     <meta
       name="viewport"
       content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
     >
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" >
+    <link rel="stylesheet" href="./style.css" >
     <title>Search bar airbnb</title>
-    <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      action="https://www.google.com/search"
+      class="form__container form__container-1"
+      method="get"
+      data-qa="big"
+    >
+      <input
+        type="text"
+        class="searchBar searchBar-big"
+        placeholder="Try &quot;Los Angeles&quot;"
+        name="search"
+        data-qa="keypress"
+      >
+      <button type="submit" class="buttom__search buttom__searchBar-big">
+        <img src="./images/Search.svg" alt="search" >
+      </button>
+    </form>
+    <form
```

Comment:
```suggestion
    </form>
    
    <form
```
don't forget to add space between one level blocks